distributed/scheduler.py

```diff
@@ -1149,7 +1130,7 @@ class TaskGroup(TaskCollection):
     __slots__ = tuple(__annotations__)
 
     def __init__(self, name: str, prefix: TaskPrefix):
-        super().__init__(name)
+        TaskCollection.__init__(self, name)
```

Comment:
`super()` is convenient and very useful in complex MROs but it also adds overhead which we don't need/want here.